addon/components/searchable-select.js

```diff
@@ -11,27 +11,39 @@ export default Ember.Component.extend({
   selected: null,
   optionLabelKey: 'title',
   optionDisabledKey: null,
+  sortBy: null,
+  limitSearchToWordBoundary: false,
+
   prompt: 'Select an option',
   searchPrompt: 'Type to search',
   noResultsMessage: 'No matching results',
-  loadingMessage: 'Searching...',
-  limitSearchToWordBoundary: false,
+
   isClearable: true,
   clearLabel: 'Clear',
-  sortBy: null,
+
+  canAddNew: false,
```

Comment:
Could this be a computed property based on if the user has passed in 'on-add'. Something like `canAddNew = (on-add !== Ember.K)`